meta-flex-distro/recipes-kernel/linux/linux-yocto_%.bbappend

```diff
@@ -0,0 +1,13 @@
+# This material contains trade secrets or otherwise confidential information owned by Siemens Industry Software Inc.
+# or its affiliates (collectively, "Siemens"), or its licensors. Access to and use of this information is strictly limited
+# as set forth in the Customer's applicable agreements with Siemens.
+# ---------------------------------------------------------------------------------------------------------------------
+# Unpublished work. Copyright 2024 Siemens
+# ---------------------------------------------------------------------------------------------------------------------
+
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+SRC_URI:append:feature-tracing = " \
```

Comment:
flex-os override should be applied here, just as you did in other recipe. If additionally you need to use tracing, you can add check with bb.utils.contains() to see if tracing is enabled in DISTRO_FEATURES.